docs/core/deploying/deploy-with-cli.md

```diff
@@ -1,201 +1,154 @@
+When you publish your app as a *framework-dependent deployment* (FDD), a `<PROJECT-NAME>.dll` file is created in the `<CURRENT-FOLDER>/bin/<BUILD-CONFIGURATION>/<TFM>/publish/` folder. To run your app, navigate to the output folder and call the `dotnet <PROJECT-NAME>.dll` command. 
 
-1. After you've debugged and tested the program, create the files to be deployed with your app for each platform that it targets.
+Your app is configured to target a specific version of .NET Core, and that runtime is required on the machine where you want to run your app. For example, if your app targets .NET Core 2.2, any machine that your app runs on must have the .NET Core 2.2 runtime installed. As stated in the [Publishing basics](#publishing-basics) section, you can edit your project file to change the default target framework or to target more than one framework.
 
-   Use the `dotnet publish` command for both target platforms as follows:
+## Framework-dependent executable
 
-      ```console
-      dotnet publish -c Release -r win10-x64
-      dotnet publish -c Release -r osx.10.11-x64
-      ```
+For the .NET Core SDK 3.x CLI, this publish mode is the default mode for the basic `dotnet publish` command. You don't need to specify any other parameters as long as want to target the current operating system.
 
-   This creates a Release (rather than a Debug) version of your app for each target platform. The resulting files are placed in a subdirectory named *publish* that's in a subdirectory of your project's *.\bin\Release\netcoreapp2.1\<runtime_identifier>* subdirectory. Note that each subdirectory contains the complete set of files (both your app files and all .NET Core files) needed to launch your app.
+By publishing a *framework-dependent executable* (FDE), a platform-specific executable is created. Similar to the FDD publish mode, your app targets the version of .NET Core that your project uses or that is specified on the command line. Your app is still created as a `.dll` file (like with FDD), but, a host executable is also generated that is specific to the target platform. You can run this executable instead of the `dotnet name.dll` command.
 
-Along with your application's files, the publishing process emits a program database (.pdb) file that contains debugging information about your app. The file is useful primarily for debugging exceptions. You can choose not to package it with your application's files. You should, however, save it in the event that you want to debug the Release build of your app.
+Your app is configured to target a specific version of .NET Core, and that runtime is required on the machine where you want to run your app. For example, if your app targets .NET Core 2.2, any machine that your app runs on must have the .NET Core 2.2 runtime installed. As stated in the [Publishing basics](#publishing-basics) section, you can edit your project file to change the default target framework or to target more than one framework.
```

Comment:
This now seems out of place -- in explaining what a FDE is, it's best to begin with similarities to FDD (of which this is one), then mention the differences (an executable that is platform-dependent and requires you to specify the target platform (or the current platform).

Reply:
@rpetrusha I've reworked the paragraph prior to this that I think does a good job explaining the differences. But I think that this restriction is the same as FDD. FDE still needs the runtime installed.